srttools/monitor/monitor.py

```diff
@@ -133,17 +159,28 @@ def __init__(
 
         # Initialize the web server, this will raise a OSError if the given
         # port is already being used
-        self._web_server = WebServer(self._extension, self._port)
+        self._web_server = WebServer(self._extension, self._localhost, self._port)
```

Comment:
The localhost option will probably only be useful in tests since pytest won't allow the use of 0.0.0.0 (any interface). This was mandatory in order to avoid what pytest would have seen as access to the internet.